_posts/2021-04-20-kubeflow-1.3-release.md

```diff
@@ -0,0 +1,145 @@
+*   Volume Manager ([Video tour](https://www.youtube.com/watch?v=jU2DtSWahdA))
+    *   The Volume Manager enables you to manage your data and persistent volumes. For the volumes in your namespace, it streamlines the creation and deletion of volumes, which then can be easily attached to your notebooks. PR [5684](https://github.com/kubeflow/kubeflow/pull/5684) 
+*   Kale ([Video tour](https://www.youtube.com/watch?v=ANBkUySirGg))
+    *   The updated Kale UI, a JupyterLab extension, simplifies your hyperparameter tuning trial set-up. The UI walks you through these steps: enter your hyperparameters as a list or a range, pick your search algorithm (Grid, Random, Bayesian) and the parameter to be optimized i.e. minimize loss. Then with a click of a button, your Katib trials are set-up, snapshotted, tracked, and run.
+*   Kubeflow Pipelines (KFP)
+    *   The KFP UI has been reorganized for a more unified experience (PR [4925](https://github.com/kubeflow/pipelines/pull/4925)), and includes the ability to manage recurring runs via new “JobsList” and “AllJobslist” pages (PR [5131](https://github.com/kubeflow/pipelines/pull/5131)) and simplified view of dependency graphs.
+
+Beyond the UIs, data scientists can also tie Notebooks with Serving more closely than ever before. In addition to the aforementioned integration with TensorBoard, Kubeflow Notebooks also now support first class deployments with TensorFlow 2.0, PyTorch, VS Code and RStudio.
+
+KFServing enhancements include simplified canary rollouts with traffic splitting at the Knative revisions level. It also delivers extended ML framework support for:
+
+*   TorchServe predict and PyTorch Captum Explain 
+*   PMMLServer, PR [1141](https://github.com/kubeflow/kfserving/pull/1141)
+*   LightGBM 
+
+## Infrastructure and operational efficiencies
+
+ML engineers will like 1.3’s delivery of operational and infrastructure efficiencies, which are coupled with streamlined installation patterns and upgraded Istio version support. The following chart provides a summary of the top production features in 1.3.  
+
+<table>
+  <tr>
+   <td><strong>Feature</strong>
+   </td>
+   <td><strong>Benefits</strong>
+   </td>
+  </tr>
+  <tr>
+   <td><a href="https://github.com/yuzliu/kfserving/blob/master/docs/MULTIMODELSERVING_GUIDE.md">Multi-model serving</a> (Alpha)
+   </td>
+   <td>More models on same infra and workaround cluster limits        i.e. # of pods & ip addresses
+   </td>
+  </tr>
+  <tr>
+   <td>Pod affinity
+   </td>
+   <td>Avoid unnecessary usage on GPU or large CPU nodes
+   </td>
+  </tr>
+  <tr>
+   <td>gRPC support 
+   </td>
+   <td>Fewer messages, less bandwidth for KFServing workloads
+   </td>
+  </tr>
+  <tr>
+   <td>Katib trial templates
+   </td>
+   <td>Simplifies hyperparameter tuning set-up for custom model types
+   </td>
+  </tr>
+  <tr>
+   <td>Katib early stopping
```

Comment:
Do we need to say that Katib support k8s>=1.18 in Kubeflow 1.3 ?

Reply:
It wouldn't hurt, I can add it, where exactly should I put it?

Reply:
@jbottum What do you think ? Is it a valuable feature that we need to mention in this Blog ?